README.md

```diff
@@ -414,11 +414,11 @@ Emitted when a data block has been downloaded.
 
 #### `feed.on('upload', index, data)`
 
-Emitted when a data block is uploaded.
+Emitted when a data block has been uploaded.
```

Comment:
I am guessing that though.

Reply:
I actually think the former is correct as it happens *before* the block is sent over the wire. See https://github.com/mafintosh/hypercore/blob/5a46735bb95e96265dea32f2919a94da120b00fa/lib/replicate.js#L181-L195

Reply:
Ok, clarified. Ty!